### PR TITLE
Added a get-group method

### LINKED ISSUE #143: GetGroup

### PR DESCRIPTION
**Pull request by hitohata**
## Overviews of implementation

Added a get-group method to the Nihsiki DB Client.

## Review points

Passed tests

### CODE REVIEW

File: backend/main/src/Shared/Adapters/DB/NishikiTableClient.ts [hunk 1]
```diff
@@ -104,6 +106,124 @@ export class NishikiDynamoDBClient {
...
+
+	/**
+	 * Save a group to the DB.
+	 * This function generates multiple putItem commands and sends it concurrently.
+	 * If user IDs are provided, this function issues [PutItem commands for creating user and group relations](https://genesis-tech-tribe.github.io/nishiki-documents/project-document/database#user).
+	 * @param groupId
+	 * @param props
+	 */
+	async saveGroup(groupId: string, props: GroupInput) {
+		const { groupName, userIds, containers } = props;
```
Comment (taniguchiiqqq) [inline backend/main/src/Shared/Adapters/DB/NishikiTableClient.ts]: This `containers` include only container ID, so its name should be containerIds like userIds.

File: backend/main/src/Shared/Adapters/DB/NishikiDBTypes.ts [hunk 2]
```diff
@@ -6,3 +6,20 @@ export type UserData = {
...
 	emailAddress: string;
 };
+
+/**
+ * Save group's props
+ */
+export type GroupInput = {
+	groupName?: string;
+	userIds?: string[];
+	containers?: string[];
```
Comment (taniguchiiqqq) [inline backend/main/src/Shared/Adapters/DB/NishikiDBTypes.ts]: I think this `containers` name should be `containerIds` like `userIds`.

**Review by taniguchiiqqq — CHANGES_REQUESTED** [review]
Thanks! LGTM with nits.

**Comment by hitohata** [comment]
@taniguchiiqqq 
Thank you. I've fixed.

**Review by taniguchiiqqq — APPROVED** [review]

**Pull request merged**